packages/volto/src/components/theme/Logo/Logo.jsx

```diff
@@ -4,13 +4,13 @@
  */
 import { defineMessages, useIntl } from 'react-intl';
 import { useEffect } from 'react';
-import { Image } from 'semantic-ui-react';
 import LogoImage from '@plone/volto/components/theme/Logo/Logo.svg';
 import { useSelector, useDispatch } from 'react-redux';
 import { Link, useLocation } from 'react-router-dom';
 import { getNavroot } from '@plone/volto/actions/navroot/navroot';
 import { flattenToAppURL, getBaseUrl } from '@plone/volto/helpers/Url/Url';
 import { hasApiExpander } from '@plone/volto/helpers/Utils/Utils';
+import Image from '@plone/volto/components/theme/Image/Image';
```

Comment:
@wesleybl Aren't we tackling this in the main PR?
Also  I found that after we upload the logo through controlpanel this fails to work as `plone_site_logo` comes as site root URL and Plone doesn't know about the prefix. 
https://github.com/plone/Products.CMFPlone/blob/6e2b2f55fc251a0e68b5ba65c09de6be867a9d39/Products/CMFPlone/utils.py#L508

Reply:
> Aren't we tackling this in the main PR?

@nileshgulia1 Yes, but I'm following @sneridagh's idea of ​​redoing PR #3592 to explain things better. So I decided to break it down into smaller, easier-to-understand PRs.

This PR practically doesn't change anything about the Volto, but it already prepares the framework for the prefix.

> Also I found that after we upload the logo through controlpanel this fails to work as plone_site_logo comes as site root URL and Plone doesn't know about the prefix.

The idea here is not yet to make the prefix work.

Reply:
@wesleybl I will put it in this agenda in upcoming volto team meeting. Also what about the links?